package.json

```diff
@@ -1,16 +1,17 @@
 {
-    "name": "crittr",
-    "version": "1.4.0",
+    "name": "@timonbandit/crittr",
```

Comment:
I can not merge this file

Reply:
Don't need to merge it :-)

Reply:
If you change it that way I can not accept the whole pull request. 
You can not change the whole urls and versions in your pull request. Adding or removing packages is needed sometimes but releasing new versions is a thing of the maintainers not the contributors. 

Do we agree?